x-pack/solutions/security/plugins/security_solution/public/asset_inventory/constants.ts

```diff
@@ -13,6 +13,7 @@ export const ASSET_INVENTORY_DATA_VIEW_ID_PREFIX = 'asset-inventory';
 export const QUERY_KEY_GRID_DATA = 'asset_inventory_grid_data';
 export const QUERY_KEY_CHART_DATA = 'asset_inventory_chart_data';
 export const QUERY_KEY_GROUPING_DATA = 'asset-inventory-grouping-data';
+export const QUERY_KEY_ASSET_INVENTORY = 'asset-inventory-query-key';
```

Comment:
Can you also include this query key in the `useFetchChartData` hook, so the chart gets refreshed as well?